src/uct/ib/base/ib_device.h

```diff
@@ -66,7 +70,8 @@ enum {
     UCT_IB_ADDRESS_FLAG_LID      = UCS_BIT(0),
     UCT_IB_ADDRESS_FLAG_IF_ID    = UCS_BIT(1),
     UCT_IB_ADDRESS_FLAG_SUBNET16 = UCS_BIT(2),
-    UCT_IB_ADDRESS_FLAG_SUBNET64 = UCS_BIT(3)
+    UCT_IB_ADDRESS_FLAG_SUBNET64 = UCS_BIT(3),
+    UCT_IB_ADDRESS_FLAG_GID_RAW  = UCS_BIT(4)
```

Comment:
UCT_IB_ADDRESS_FLAG_GID ("raw" is only when it's inside the union. the correct terminology is just a "gid")